floodviz/views.py

```diff
@@ -1,7 +1,112 @@
 from flask import request, render_template, abort
+import csv
+import requests
+import yaml
+import os
 
 from . import app
 
+
 @app.route('/home/')
 def home():
-    return render_template('index.html')
\ No newline at end of file
+    return render_template('index.html')
+
+
+# This program requires svgis to be installed (which in turn requires Fiona and GDAL)
```

Comment:
svgis may be simpler to use but there are also additional libraries that have to be installed. I think it's something we will need to talk about with the data science folks. It will be interesting to see the SVG that is produced by this project. We will talk more about this at the demo tomorrow.